### PR TITLE
docs: add an example how to replace the DefaultHttpUrlGenerator

### PR DESCRIPTION
**Pull request by bert2**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

N/A

Closes N/A

## What is the new behavior?

N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[X] No
```

## Other information

N/A

### CODE REVIEW

File: projects/ngrx.io/content/guide/data/extension-points.md [hunk 1]
```diff
@@ -167,6 +167,96 @@ export const selectedCatsWithOwners = createSelector(
...
+        collectionResourceUrl: url
+      };
+      this.registerHttpResourceUrls({ [entityName]: resourceUrls });
+    }
+    return resourceUrls;
+  }
+}
+```
+
+In order to hook it into ngrx data we need to provide it in the module where ngrx data is imported:
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/data/extension-points.md]:
```suggestion
Override the `HttpUrlGenerator` provider in the root `AppModule` where `EntityDataModule.forRoot()` is imported:
```

File: projects/ngrx.io/content/guide/data/extension-points.md [hunk 2]
```diff
@@ -167,6 +167,96 @@ export const selectedCatsWithOwners = createSelector(
...
+  ],
+  providers: [
+    // ...
+    { provide: HttpUrlGenerator, useClass: PluralHttpUrlGenerator }
+  ]
+})
+export class AppModule {}
+```
+
+Don't forget the unit tests:
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/data/extension-points.md]:
```suggestion
To unit test the custom HTTP URL generator:
```

File: projects/ngrx.io/content/guide/data/extension-points.md (hunk 1 above)
Comment (bert2) [reply projects/ngrx.io/content/guide/data/extension-points.md]: LGTM, but I'm not sure if I need to update the branch. Or will they come through automatically, when the change request is resolved?

Comment (brandonroberts) [reply projects/ngrx.io/content/guide/data/extension-points.md]: If you accept the suggestions, it will update the branch

Comment (brandonroberts) [reply projects/ngrx.io/content/guide/data/extension-points.md]: If you accept the suggestions, it will update the branch

**Comment by brandonroberts** [comment]
Thanks @bert2!

**Pull request merged**